### PR TITLE
added info and pics for betty and tiana

### PR DESCRIPTION
**Pull request by betty**
added pics and info for Tatiana and Blagovesta
please review and merge

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -296,7 +296,67 @@
...
+                            <a href="https://github.com/blagovesta"
+                               rel="noreferrer"
+                               target="_blank">
+                                <img class="icon"
+                                     src="images/icons/github.svg"
+                                     alt="GitHub" />
+                            </a>
+                        </div>
+                        <span>
+                            A mix breed of an undercover web developer and a startup aficionado. Likes reserve engineering people and secure systems.
```
Comment (SolalPirelli) [inline index.html]: typo -> should be "reverse-engineering", no?

File: index.html [hunk 2]
```diff
@@ -316,30 +376,30 @@
...
                                target="_blank">
                                 <img class="icon"
                                      src="images/icons/github.svg"
                                      alt="GitHub" />
-                            </a-->
+                            </a>
                         </div>
                         <span>
-                            <!-- to fill -->
+                           Analyst. Interested in data science, system modeling and project management
```
Comment (SolalPirelli) [inline index.html]: end this with a period

File: index.html [hunk 3]
```diff
@@ -492,7 +552,7 @@
     </section>
 
     <section class="people alumni">
-        <h1>Alumnis</h1>
+        <h1>Alumni</h1>
```
Comment (SolalPirelli) [inline index.html:555]: 👍

File: index.html (hunk 1 above)
Comment (betty) [reply index.html]: googled it for a while and found it in both forms
https://en.wikipedia.org/wiki/Reverse_engineering

File: index.html (hunk 2 above)
Comment (betty) [reply index.html]: at least three dots... may result in an exclamation mark!

**Pull request merged**